lib/screens/delivery_privacy_policy_page.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:badiup/widgets/banner_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import '../colors.dart';
+import '../sign_in.dart';
+import '../widgets/cart_button.dart';
+
+class DeliveryPrivacyPolicy extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _buildAppBar(context),
+      body: _buildBody(context),
+      bottomNavigationBar: BackToProductListBannerButton(context: context),
+    );
+  }
+
+  Widget _buildAppBar(BuildContext context) {
+    return AppBar(
+      elevation: 0.0,
+      backgroundColor: paletteLightGreyColor,
+      iconTheme: IconThemeData(color: paletteBlackColor),
+      actions: <Widget>[
+        currentSignedInUser.isAdmin() ? Container() : CartButton(),
+      ],
+    );
+  }
+
+  Widget _buildBody(BuildContext context) {
+    return Stack(
+      children: <Widget>[
+        Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 16.0),
+          child: ListView(children: <Widget>[
+            SizedBox(height: 17),
+            _buildDeliveryPrivacyPolicyTitle(),
+            SizedBox(height: 17),
+            Divider(thickness: 1.0, color: kPaletteBorderColor),
+            SizedBox(height: 7),
+            _buildDeliveryPrivacyPolicyContent(),
+            SizedBox(height: 37),
+          ]),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildDeliveryPrivacyPolicyTitle() {
+    return Row(mainAxisAlignment: MainAxisAlignment.start, children: <Widget>[
+      Text(
+        "プライバシーポリシー",
+        style: TextStyle(
+          fontSize: 16,
+          color: paletteBlackColor,
+          fontWeight: FontWeight.w600,
+        ),
+      ),
+    ]);
+  }
+
+  Widget _buildDeliveryPrivacyPolicyContent() {
+    return Column(
+      children: <Widget>[
+        Row(children: [
+          Expanded(
+            child: Text(
+              "特定非営利活動法人ゴスペルエイド（以下「当団体」）は、以下のとおり個人情報保護方針を定め、個人情報保護の仕組みを構築し、全社員に個人情報保護の重要性の認識と取組みを徹底させることにより、個人情報の保護を推進致します。",
```

Comment:
Is it possible to store this text in the database and fetch it from there? This gives us flexibility to modify this text whenever needed.

Reply:
I did it by the first, however, I'm not able to give format when getting from the database. I could save it in parts if is necessary to bring it from the data base

Reply:
Yes, you can save it in parts in database. You can see here for a sample implementation: https://github.com/GospelAid/badiup/blob/master/lib/screens/privacy_policy_page.dart#L87